WooCommerce/Classes/ViewRelated/ReusableViews/OverlayMessageView.xib

```diff
@@ -28,8 +26,8 @@
                             <nil key="textColor"/>
                             <nil key="highlightedColor"/>
                         </label>
-                        <button opaque="NO" contentMode="scaleToFill" contentHorizontalAlignment="center" contentVerticalAlignment="center" buttonType="roundedRect" lineBreakMode="middleTruncation" translatesAutoresizingMaskIntoConstraints="NO" id="JeL-xN-0dT">
-                            <rect key="frame" x="172.5" y="210.5" width="30" height="30"/>
+                        <button opaque="NO" contentMode="scaleToFill" contentHorizontalAlignment="center" contentVerticalAlignment="center" lineBreakMode="middleTruncation" translatesAutoresizingMaskIntoConstraints="NO" id="JeL-xN-0dT">
+                            <rect key="frame" x="172.5" y="210.5" width="30" height="34"/>
```

Comment:
Same here

Reply:
same as above, I updated the UIButton type from `system` to `custom` in line 31 - I thought about only committing the diff of actual change but I feel like it's going to be generated again whenever we open the xib 😐

Reply:
lemme know if you prefer not committing the auto-generated random diffs in xib! it might help with review, just that we have to discard these diffs

Reply:
It's better to let Xcode auto-update the the .xib files. If you're opening it to make edits, it's expected and totally fine to let it change the lines.